packages/module/src/MessageBox/MessageBox.tsx

```diff
@@ -72,6 +78,9 @@ const MessageBox: React.FunctionComponent<MessageBoxProps> = ({ children, classN
       <JumpButton position="top" isHidden={isOverflowing && atTop} onClick={scrollToTop} />
       <div className={`pf-chatbot__messagebox ${className ?? ''}`} ref={messageBoxRef}>
         {children}
+        <div className="pf-chatbot__messagebox-announcement" aria-live="polite">
```

Comment:
If this always exists on the page, I wonder if it could be set programatically (if an announcement is not passed by default. Like could we provide an out of the box default that reads out the new announcement?)

I think that could be a resonable follow up for the future.